woocommerce-gutenberg-products-block.php

```diff
@@ -261,3 +261,54 @@ function woocommerce_blocks_get_php_translation_from_core( $translation, $text,
 }
 
 add_filter( 'gettext', 'woocommerce_blocks_get_php_translation_from_core', 10, 3 );
+
+/**
+ * Check the current WC Blocks path. If the WC Blocks plugin is active but the
+ * current path is from the WC Core, we can consider the plugin is outdated
+ * because JetPack autoloader always loads the newer package.
```

Comment:
Not a big deal, but I think the correct branding is as follows:
```suggestion
 * because Jetpack Autoloader always loads the newer package.
```